apps/src/templates/projects/ProjectCard.jsx

```diff
@@ -35,9 +205,217 @@ export default class ProjectCard extends React.Component {
+                />
+              </div>
+            ) : (
+              <div>
+                <div
+                  style={{
+                    display: 'flex',
+                    justifyContent: 'space-between',
+                    alignItems: 'center',
+                  }}
+                >
+                  <h3 style={{margin: 0}}>Report Abuse</h3>
+                  <button
+                    type="reset"
+                    onClick={this.cancel}
+                    style={{
+                      ...styles.transparentButton,
+                      width: 24,
+                      height: 24,
+                      fontSize: 16,
+                      padding: 5,
+                    }}
+                  >
+                    <FontAwesome icon="x" style={{color: '#D4D5D7'}} />
+                  </button>
+                </div>
+                <hr style={styles.popUpLines} />
+                <p style={styles.popUpBody}>
+                  Why are you reporting this content?
+                </p>
+                <div>
+                  {checkboxes.map(checkbox => (
+                    <label
+                      key={checkbox.name}
+                      style={{
+                        display: 'flex',
+                        alignItems: 'center',
+                        marginTop: 5,
+                        marginBottom: 5,
+                      }}
+                    >
+                      <input
+                        type="checkbox"
+                        checked={checkbox.checked}
+                        onChange={() =>
+                          this.handleCheckboxChange(checkbox.name)
+                        }
+                        style={{
+                          marginRight: 10,
+                          marginTop: 0,
+                          accentColor: '#0093A4',
+                        }}
+                      />
+                      <span
+                        style={{
+                          flex: '1',
+                          ...styles.popUpBody,
+                          verticalAlign: 'middle',
+                        }}
+                      >
+                        {checkbox.name.replace(/-/g, ' ')}
+                      </span>
+                    </label>
+                  ))}
+                </div>
+                {isAnyCheckboxSelected ? (
+                  <div
+                    style={{padding: 5}}
+                    className="g-recaptcha"
+                    data-sitekey={captchaSiteKey}
+                    data-callback="onCaptchaSubmit"
+                    data-expired-callback="onCaptchaExpired"
+                  />
+                ) : null}
+                <hr style={styles.popUpLines} />
+                <div style={{display: 'flex', justifyContent: 'flex-end'}}>
+                  <Button
+                    onClick={this.cancel}
+                    text={'Cancel'}
+                    color={Button.ButtonColor.white}
```

Comment:
use `Button.ButtonColor.neutralDark` here and you please delete those inline styles for this button (you won't need them)